presto-docs/src/main/sphinx/release/release-0.157.1-t.rst

```diff
@@ -264,3 +264,29 @@ The following has been added to 0.157.1-t.0.5:
 * Fix incorrect results for UNION ALL queries with duplicate column names.
   `https://github.com/prestodb/presto/issues/6935 <https://github.com/facebook/presto/issues/6935>`_
 
+=====================
+Release 0.157.1-t.0.6
+=====================
+
+The following has been added to 0.157.1-t.0.6:
+
+* Fix ``ConnectorMetadata#beginQuery`` behaviour. Now it is being called before ``ConnectorSplitManager#getSplits``
+
+=====================
+Release 0.157.1-t.0.7
+=====================
+
+The following has been added to 0.157.1-t.0.7:
+
+* Fix incorrect pruning of join output columns
+
+=====================
+Release 0.157.1-t.0.8
+=====================
+
+The following has been added to 0.157.1-t.0.8:
+
+* Fix creation of Hive external tables.
+  `https://github.com/prestodb/presto/pull/6813 <https://github.com/prestodb/presto/pull/6813>`_
+* Fix HDFS configuration resources processing in Hive connector. This might have caused problems with connecting to HA HDFS.
```

Comment:
Suggest:
Fix HDFS configuration loading in the Hive connector. Without this fix, there sometimes were problems with connecting to HA HDFS.